### DISCUSSION TITLE
Emoji icon

### DISCUSSION THREAD

**Discussion by cdpestitschek** [twilio-labs/paste#2499] [Request: Icon] [answered]
Ahoy Paste Team 👋 

**Background**
We are working on a plugin that enables the use of emojis in chat. At the moment we need an icon to give access to this library/list of emojis, this icon will be displayed in the input message component. 
<img width="372" alt="Group 302" src="https://user-images.githubusercontent.com/100429369/176160584-30618df6-b871-4156-b313-9eb5ceda9164.png">

**Icon Validation**
1) Does this one meet paste standards?
<img width="20" alt="Vector" src="https://user-images.githubusercontent.com/100429369/176160267-8c4ad21d-276a-4868-a5a9-92d6254e3eda.png">

If not...
2) Can the Paste team create an emoji for us?
or
3)Should we use this one from Streamline?
https://www.streamlinehq.com/icons/ui-line-pro?category=&search=smil

🧰  Assets and informations:
**Stage of design:** High Fidelity
**Links to mockups or prototype:**[ Emoji picker - Figma file](https://www.figma.com/file/RaOJnlbBLv4eeA1AvPppC6/Emoji-Picker?node-id=25%3A20436)

Looking forward to receive feedback on this :)

**Comment by TheSisb** [comment, marked as answer]
Hi @cdpestitschek, thanks for reaching out!

To unblock you, we'd recommend using this free streamline icon https://www.streamlinehq.com/icons/streamline-mini-line/mail/smiley/mail-smiley-happy-face.
In the future if we do add something to Paste, it will likely be based on that icon.

**Comment by cdpestitschek** [comment]
Hi @TheSisb

Thanks for the reply. Done!
